mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowTrackingContext.java

```diff
@@ -0,0 +1,59 @@
+package org.mlflow.tracking;
+
+import org.mlflow.api.proto.Service.*;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class MlflowTrackingContext {
```

Comment:
This is kind of a long name; maybe just MLflowContext? Though I guess that could be confusingly similar to MLflowClient for some people.